gradle.properties

```diff
@@ -16,3 +16,9 @@ org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8 \
 android.useAndroidX=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+
+# Version of JDK to use for running tests.
+# Possible values:
+# - auto -> use Java running Gradle, best for local dev.
+# - A Java major version number, like 11 or 17.
+mockito.test.java=auto
```

Comment:
Could be an env var, but prop can be documented nicely here and also have a global default.